### PR TITLE
linux/a*: add Spanish translation

### PR DESCRIPTION
**Pull request by Jaaq5**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/linux/a2disconf.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# a2disconf
+
+> Deshabilite un archivo de configuración de Apache en sistemas operativos basados en Debian.
```
Comment (navarroaxel) [inline pages.es/linux/a2disconf.md:3]:
```suggestion
> Deshabilita un archivo de configuración de Apache en sistemas operativos basados en Debian.
```

File: pages.es/linux/a2disconf.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+> Más información: <https://manpages.debian.org/latest/apache2/a2disconf.8.en.html>.
+
+- Deshabilitar un archivo de configuración:
```
Comment (navarroaxel) [inline pages.es/linux/a2disconf.md:6]: For Romance languages, the present indicative fits better.
```suggestion
- Deshabilita un archivo de configuración:
```

File: pages.es/linux/a2disconf.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+`sudo a2disconf {{archivo_de_configuración}}`
+
+- No mostrar mensajes informativos:
```
Comment (navarroaxel) [inline pages.es/linux/a2disconf.md:10]:
```suggestion
- No muestra mensajes informativos:
```

File: pages.es/linux/a2dismod.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# a2dismod
+
+> Deshabilite un módulo de Apache en sistemas operativos basados en Debian.
```
Comment (navarroaxel) [inline pages.es/linux/a2dismod.md:3]:
```suggestion
> Deshabilita un módulo de Apache en sistemas operativos basados en Debian.
```

File: pages.es/linux/a2dismod.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+> Más información: <https://manpages.debian.org/latest/apache2/a2dismod.8.en.html>.
+
+- Deshabilitar un módulo:
```
Comment (navarroaxel) [inline pages.es/linux/a2dismod.md:6]:
```suggestion
- Deshabilita un módulo:
```

File: pages.es/linux/a2dismod.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+`sudo a2dismod {{módulo}}`
+
+- No mostrar mensajes informativos:
```
Comment (navarroaxel) [inline pages.es/linux/a2dismod.md:10]:
```suggestion
- No muestra mensajes informativos:
```

File: pages.es/linux/a2dissite.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
+# a2dissite
+
+> Deshabilite un servidor virtual Apache en sistemas operativos basados en Debian.
+> Más información: <https://manpages.debian.org/latest/apache2/a2dissite.8.en.html>.
+
+- Deshabilitar un host virtual:
```
Comment (navarroaxel) [inline pages.es/linux/a2dissite.md:6]:
```suggestion
- Deshabilita un host virtual:
```

File: pages.es/linux/a2dissite.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
+# a2dissite
+
+> Deshabilite un servidor virtual Apache en sistemas operativos basados en Debian.
```
Comment (navarroaxel) [inline pages.es/linux/a2dissite.md:3]:
```suggestion
> Deshabilita un servidor virtual Apache en sistemas operativos basados en Debian.
```

File: pages.es/linux/a2dissite.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+> Más información: <https://manpages.debian.org/latest/apache2/a2dissite.8.en.html>.
+
+- Deshabilitar un host virtual:
+
+`sudo a2dissite {{host_virtual}}`
+
+- No mostrar mensajes informativos:
```
Comment (navarroaxel) [inline pages.es/linux/a2dissite.md:10]:
```suggestion
- No muestra mensajes informativos:
```

File: pages.es/linux/a2enconf.md [hunk 10]
```diff
@@ -0,0 +1,12 @@
+# a2enconf
+
+> Habilite un archivo de configuración de Apache en sistemas operativos basados en Debian.
```
Comment (navarroaxel) [inline pages.es/linux/a2enconf.md:3]:
```suggestion
> Habilita un archivo de configuración de Apache en sistemas operativos basados en Debian.
```

File: pages.es/linux/a2enconf.md [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+> Más información: <https://manpages.debian.org/latest/apache2/a2enconf.8.en.html>.
+
+- Habilitar un archivo de configuración:
```
Comment (navarroaxel) [inline pages.es/linux/a2enconf.md:6]:
```suggestion
- Habilita un archivo de configuración:
```

File: pages.es/linux/a2enconf.md [hunk 12]
```diff
@@ -0,0 +1,12 @@
...
+
+`sudo a2enconf {{archivo_de_configuración}}`
+
+- No mostrar mensajes informativos:
```
Comment (navarroaxel) [inline pages.es/linux/a2enconf.md:10]:
```suggestion
- No muestra mensajes informativos:
```

File: pages.es/linux/a2enmod.md [hunk 13]
```diff
@@ -0,0 +1,12 @@
+# a2enmod
+
+> Habilite un módulo de Apache en sistemas operativos basados en Debian.
```
Comment (navarroaxel) [inline pages.es/linux/a2enmod.md:3]:
```suggestion
> Habilita un módulo de Apache en sistemas operativos basados en Debian.
```

File: pages.es/linux/a2enmod.md [hunk 14]
```diff
@@ -0,0 +1,12 @@
...
+> Más información: <https://manpages.debian.org/latest/apache2/a2enmod.8.en.html>.
+
+- Habilitar un módulo:
```
Comment (navarroaxel) [inline pages.es/linux/a2enmod.md:6]:
```suggestion
- Habilita un módulo:
```

File: pages.es/linux/a2enmod.md [hunk 15]
```diff
@@ -0,0 +1,12 @@
...
+
+`sudo a2enmod {{módulo}}`
+
+- No mostrar mensajes informativos:
```
Comment (navarroaxel) [inline pages.es/linux/a2enmod.md:10]:
```suggestion
- No muestra mensajes informativos:
```

File: pages.es/linux/a2ensite.md [hunk 16]
```diff
@@ -0,0 +1,12 @@
+# a2ensite
+
+> Habilite un servidor virtual Apache en sistemas operativos basados en Debian.
+> Más información: <https://manpages.debian.org/latest/apache2/a2ensite.8.en.html>.
+
+- Habilitar un host virtual:
```
Comment (navarroaxel) [inline pages.es/linux/a2ensite.md:6]:
```suggestion
- Habilita un host virtual:
```

File: pages.es/linux/a2ensite.md [hunk 17]
```diff
@@ -0,0 +1,12 @@
+# a2ensite
+
+> Habilite un servidor virtual Apache en sistemas operativos basados en Debian.
```
Comment (navarroaxel) [inline pages.es/linux/a2ensite.md:3]:
```suggestion
> Habilita un servidor virtual Apache en sistemas operativos basados en Debian.
```

File: pages.es/linux/a2ensite.md [hunk 18]
```diff
@@ -0,0 +1,12 @@
...
+> Más información: <https://manpages.debian.org/latest/apache2/a2ensite.8.en.html>.
+
+- Habilitar un host virtual:
+
+`sudo a2ensite {{host_virtual}}`
+
+- No mostrar mensajes informativos:
```
Comment (navarroaxel) [inline pages.es/linux/a2ensite.md:10]:
```suggestion
- No muestra mensajes informativos:
```

File: pages.es/linux/a2query.md [hunk 19]
```diff
@@ -0,0 +1,24 @@
+# a2query
+
+> Recuperar la configuración del tiempo de ejecución de Apache en sistemas operativos basados en Debian.
```
Comment (navarroaxel) [inline pages.es/linux/a2query.md:3]:
```suggestion
> Recupera la configuración en tiempo de ejecución de Apache en sistemas operativos basados en Debian.
```

File: pages.es/linux/a2query.md [hunk 20]
```diff
@@ -0,0 +1,24 @@
...
+> Más información: <https://manpages.debian.org/latest/apache2/a2query.html>.
+
+- Lista de módulos de Apache habilitados:
```
Comment (navarroaxel) [inline pages.es/linux/a2query.md:6]:
```suggestion
- Lista módulos de Apache habilitados:
```

File: pages.es/linux/a2query.md [hunk 21]
```diff
@@ -0,0 +1,24 @@
...
+
+`sudo a2query -m`
+
+- Comprobar si un módulo específico está instalado:
```
Comment (navarroaxel) [inline pages.es/linux/a2query.md:10]:
```suggestion
- Comprueba si un módulo específico está instalado:
```

File: pages.es/linux/a2query.md [hunk 22]
```diff
@@ -0,0 +1,24 @@
...
+
+`sudo a2query -m {{nombre_del_módulo}}`
+
+- Lista de hosts virtuales habilitados:
```
Comment (navarroaxel) [inline pages.es/linux/a2query.md:14]:
```suggestion
- Lista hosts virtuales habilitados:
```

File: pages.es/linux/a2query.md [hunk 23]
```diff
@@ -0,0 +1,24 @@
...
+
+`sudo a2query -s`
+
+- Mostrar el Módulo de Procesamiento Múltiple habilitado actualmente:
```
Comment (navarroaxel) [inline pages.es/linux/a2query.md:18]:
```suggestion
- Muestra el Módulo de Procesamiento Múltiple actualmente habilitado:
```

File: pages.es/linux/a2query.md [hunk 24]
```diff
@@ -0,0 +1,24 @@
...
+
+`sudo a2query -M`
+
+- Mostrar la versión de Apache:
```
Comment (navarroaxel) [inline pages.es/linux/a2query.md:22]:
```suggestion
- Muestra la versión de Apache:
```

File: pages.es/linux/abbr.md [hunk 25]
```diff
@@ -0,0 +1,21 @@
+# abbr
+
+> Administrar abreviaturas para el shell fish.
```
Comment (navarroaxel) [inline pages.es/linux/abbr.md:3]:
```suggestion
> Administra abreviaturas para el shell fish.
```

File: pages.es/linux/abbr.md [hunk 26]
```diff
@@ -0,0 +1,21 @@
...
+> Las palabras definidas por el usuario se reemplazan con frases más largas después de ingresarlas.
+> Más información: <https://fishshell.com/docs/current/cmds/abbr.html>.
+
+- Añadir una nueva abreviatura:
```
Comment (navarroaxel) [inline pages.es/linux/abbr.md:7]:
```suggestion
- Añade una nueva abreviatura:
```

File: pages.es/linux/abbr.md [hunk 27]
```diff
@@ -0,0 +1,21 @@
...
+
+`abbr --add {{nombre_abreviatura}} {{comando}} {{argumentos_del_comando}}`
+
+- Cambiar el nombre de una abreviatura existente:
```
Comment (navarroaxel) [inline pages.es/linux/abbr.md:11]:
```suggestion
- Cambia el nombre de una abreviatura existente:
```

File: pages.es/linux/abbr.md [hunk 28]
```diff
@@ -0,0 +1,21 @@
...
+
+`abbr --rename {{nombre_antiguo}} {{nombre_nuevo}}`
+
+- Borrar una abreviatura existente:
```
Comment (navarroaxel) [inline pages.es/linux/abbr.md:15]:
```suggestion
- Borra una abreviatura existente:
```

File: pages.es/linux/abbr.md [hunk 29]
```diff
@@ -0,0 +1,21 @@
...
+
+`abbr --erase {{nombre_abreviatura}}`
+
+- Importar las abreviaturas definidas en otro host a través de SSH:
```
Comment (navarroaxel) [inline pages.es/linux/abbr.md:19]:
```suggestion
- Importa las abreviaturas definidas en otro host a través de SSH:
```

File: pages.es/linux/abroot.md [hunk 30]
```diff
@@ -0,0 +1,33 @@
...
+
+> La utilidad ABRoot proporciona inmutabilidad y atomicidad completas al realizar transacciones entre 2 estados de partición raíz (A⟺B).
+> También permite transacciones bajo demanda a través de un shell transaccional.
+> Más información: <https://github.com/Vanilla-OS/ABRoot>.
+
+- Muestra el estado actual o futuro de la partición raíz:
+
+`sudo abroot get {{present|future}}`
+
+- Ingrese el shell transaccional en la partición raíz futura y cambie la raíz en el próximo arranque:
```
Comment (navarroaxel) [inline pages.es/linux/abroot.md:11]:
```suggestion
- Ingresa el shell transaccional en la partición raíz futura y cambia la raíz en el próximo arranque:
```

File: pages.es/linux/abroot.md [hunk 31]
```diff
@@ -0,0 +1,33 @@
...
+
+`sudo abroot shell`
+
+- Ejecute un comando específico en el shell transaccional en la futura partición raíz y cambie a él en el siguiente arranque:
```
Comment (navarroaxel) [inline pages.es/linux/abroot.md:15]:
```suggestion
- Ejecuta un comando específico en el shell transaccional en la futura partición raíz y cambia a él en el siguiente arranque:
```

File: pages.es/linux/abroot.md [hunk 32]
```diff
@@ -0,0 +1,33 @@
...
+
+`sudo abroot exec "{{comando}}"`
+
+- Instale paquetes específicos en el host dentro del shell transaccional en la partición raíz futura y cambie a él en el próximo arranque:
```
Comment (navarroaxel) [inline pages.es/linux/abroot.md:19]:
```suggestion
- Instala paquetes específicos en el host dentro del shell transaccional en la partición raíz futura y cambia a él en el próximo arranque:
```

File: pages.es/linux/abroot.md [hunk 33]
```diff
@@ -0,0 +1,33 @@
...
+
+`sudo abroot exec apt install {{paquete1 paquete2 ...}}`
+
+- Actualice la partición de arranque (solo para usuarios avanzados):
```
Comment (navarroaxel) [inline pages.es/linux/abroot.md:23]:
```suggestion
- Actualiza la partición de arranque (solo para usuarios avanzados):
```

File: pages.es/linux/abroot.md [hunk 34]
```diff
@@ -0,0 +1,33 @@
...
+
+`sudo abroot _update-boot`
+
+- Mostrar ayuda:
```
Comment (navarroaxel) [inline pages.es/linux/abroot.md:27]:
```suggestion
- Muestra la ayuda:
```

File: pages.es/linux/abroot.md [hunk 35]
```diff
@@ -0,0 +1,33 @@
...
+
+`abroot --help`
+
+- Mostrar version:
```
Comment (navarroaxel) [inline pages.es/linux/abroot.md:31]:
```suggestion
- Muestra la version:
```

File: pages.es/linux/ac.md [hunk 36]
```diff
@@ -0,0 +1,24 @@
...
+
+- Imprime cuánto tiempo han estado conectados los usuarios en horas:
+
+`ac --individual-totals`
+
+- Imprime cuánto tiempo ha estado conectado un usuario en particular en horas:
+
+`ac --individual-totals {{nombre_usuario}}`
+
+- Imprimir cuánto tiempo un usuario en particular ha estado conectado en horas por día (en total):
```
Comment (navarroaxel) [inline pages.es/linux/ac.md:18]:
```suggestion
- Imprime cuánto tiempo un usuario en particular ha estado conectado en horas por día (en total):
```

File: pages.es/linux/ac.md [hunk 37]
```diff
@@ -0,0 +1,24 @@
+# ac
+
+> Imprimir estadísticas sobre cuánto tiempo han estado conectados los usuarios.
```
Comment (navarroaxel) [inline pages.es/linux/ac.md:3]:
```suggestion
> Imprime estadísticas sobre cuánto tiempo han estado conectados los usuarios.
```

File: pages.es/linux/ac.md [hunk 38]
```diff
@@ -0,0 +1,24 @@
...
+
+- Imprime cuánto tiempo ha estado conectado un usuario en particular en horas:
+
+`ac --individual-totals {{nombre_usuario}}`
+
+- Imprimir cuánto tiempo un usuario en particular ha estado conectado en horas por día (en total):
+
+`ac --daily-totals --individual-totals {{nombre_usuario}}`
+
+- Mostrar también detalles adicionales:
```
Comment (navarroaxel) [inline pages.es/linux/ac.md:22]:
```suggestion
- Muestra también detalles adicionales:
```

**Comment by Jaaq5** [comment]
Suggestions applied

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**